src/current/cockroachcloud/authorization.md

```diff
@@ -48,7 +44,7 @@ The following CockroachDB {{ site.data.products.cloud }} organization roles can
 
 ### Organization Member
 
-This default role is granted to all organization users once they are invited. It grants no permissions to perform cluster or organization actions.
+This default role is granted to all organization users when they are invited or provisioned. It grants no permissions to perform cluster or organization actions.
```

Comment:
suggest adding links to:

- roles
- permissions
- cluster or org actions

since as a user I could get a header link to this section and wonder "what are these things"

Reply:
This is a subsection of the section about roles, so a link there would be redundant. I rewrote a bit so the others are no longer needed, but I did link to how to edit a user's roles.